### ISSUE TITLE
Prepend `super::` to function call?

### ISSUE THREAD

**Issue by jhpratt** [la10736/rstest#173]
When seeing if I could make reasonable use of `rstest` for `time`, I quickly ran into an issue. If a case parameter is the same as the function name, compilation will fail. For example,

```rust
#[rstest]
#[case(Sunday, Monday)]
fn next(#[case] day: Weekday, #[case] next: Weekday) {
    assert_eq!(day.next(), next);
}
```

This fails because `next` is both a case parameter and the function name, and it is ambiguous in the expansion:

```rust
#[cfg(test)]
fn next(day: Weekday, next: Weekday) {
    assert_eq!(day.next(), next);
}
#[cfg(test)]
mod next {
    use super::*;
    #[test]
    fn case_1() {
        let day = Sunday;
        let next = Monday;
        next(day, next)
    }
}
```

However, if `next(day, next)` is instead `super::next(day, next)`, compilation would succeed. Given that the module is generated by `rstest`, we know that this path is always correct. Would it be possible to add this prefix to support the edge case?

Thanks for maintaining this crate!

**Comment by la10736** [comment]
Yeah!!! Good point! I never thought about it. I'll fix it as soon as possible but I'm quite slow at the moment ... I've no too much time to spend on `rstest` in this period :cry:
If you can submit a PR I'll really appreciate it :smile:

**Comment by jhpratt** [comment]
I'm the same way, really. Lots going on! No rush on my end, naturally.
